### DISCUSSION TITLE
Why pixijs doesnt allow keyboard events  ?

### DISCUSSION THREAD

**Discussion by danilockthar** [pixijs/pixijs#10661] [General]
I was trying to add an event listener to the app.canvas element, something like 'keydown' but it didn't work, then when i look at the documentation i find out that keyboard events are not available.

https://pixijs.com/8.x/guides/components/interaction?_highlight=event#event-types

Why is this ?

**Comment by bigtimebuddy** [comment]
You can add keyboard events to the `document`, for example:
https://pixiplayground.com/#/edit/7p8j07CvmuOhCPKLsnta7

Keyboard events are typically added to the document or can also be added to textarea and input elements (not the canvas). Pixi does not consider keyboard events interaction events (pointer, touch or mouse).

Hope that helps.
